pipeline/genomeBrowserTrack/brcaExchange.as

```diff
@@ -0,0 +1,142 @@
+    lstring Allele_frequency_AMR_ExAC; "Allele frequency AMR ExAC"
+    lstring BX_ID_BIC; "BX ID BIC"
+    lstring Patient_nationality_BIC; "Patient nationality BIC"
+    lstring Clinical_importance_BIC; "Clinical importance BIC"
+    lstring Clinical_classification_BIC; "Clinical classification BIC"
+    lstring Literature_citation_BIC; "Literature citation BIC"
+    lstring Number_of_family_member_carrying_mutation_BIC; "Number of family member carrying mutation BIC"
+    lstring Germline_or_Somatic_BIC; "Germline or Somatic BIC"
+    lstring Ethnicity_BIC; "Ethnicity BIC"
+    lstring Mutation_type_BIC; "Mutation type BIC"
+    lstring IARC_class_exLOVD; "IARC class exLOVD"
+    lstring Sum_family_LR_exLOVD; "Sum family LR exLOVD"
+    lstring Combined_prior_probablility_exLOVD; "Combined prior probablility exLOVD"
+    lstring BX_ID_exLOVD; "BX ID exLOVD"
+    lstring Literature_source_exLOVD; "Literature source exLOVD"
+    lstring Co_occurrence_LR_exLOVD; "Co occurrence LR exLOVD"
+    lstring Posterior_probability_exLOVD; "Posterior probability exLOVD"
+    lstring Missense_analysis_prior_probability_exLOVD; "Missense analysis prior probability exLOVD"
+    lstring Segregation_LR_exLOVD; "Segregation LR exLOVD"
+    lstring Polyphen_Prediction; "Polyphen Prediction"
+    lstring Polyphen_Score; "Polyphen Score"
+    lstring Sift_Score; "Sift Score"
+    lstring Sift_Prediction; "Sift Prediction"
+    lstring Reference_Sequence; "Reference Sequence"
+    lstring BIC_Nomenclature; "BIC Nomenclature"
+    lstring HGVS_cDNA; "HGVS cDNA"
+    lstring Gene_Symbol; "Gene Symbol"
+    lstring HGVS_Protein; "HGVS Protein"
+    lstring Protein_Change; "Protein Change"
+    lstring Genomic_Coordinate_hg38; "Genomic Coordinate hg38"
+    lstring Hg38_Start; "Hg38 Start"
+    lstring Hg38_End; "Hg38 End"
+    lstring Hg37_Start; "Hg37 Start"
+    lstring Hg37_End; "Hg37 End"
+    lstring Hg36_Start; "Hg36 Start"
+    lstring Hg36_End; "Hg36 End"
+    lstring HGVS_RNA; "HGVS RNA"
+    lstring Allele_Frequency; "Allele Frequency"
+    lstring Max_Allele_Frequency; "Max Allele Frequency"
+    lstring Genomic_Coordinate_hg37; "Genomic Coordinate hg37"
+    lstring Genomic_Coordinate_hg36; "Genomic Coordinate hg36"
+    lstring Source_URL; "Source URL"
+    lstring Discordant; "Discordant"
+    lstring Synonyms; "Synonyms"
+    lstring Pathogenicity_expert; "Pathogenicity expert"
+    lstring Pathogenicity_all; "Pathogenicity all"
+    lstring pyhgvs_Genomic_Coordinate_36; "pyhgvs Genomic Coordinate 36"
+    lstring pyhgvs_Genomic_Coordinate_37; "pyhgvs Genomic Coordinate 37"
+    lstring pyhgvs_Genomic_Coordinate_38; "pyhgvs Genomic Coordinate 38"
+    lstring pyhgvs_Hg36_Start; "pyhgvs Hg36 Start"
+    lstring pyhgvs_Hg36_End; "pyhgvs Hg36 End"
+    lstring pyhgvs_cDNA; "pyhgvs cDNA"
+    lstring pyhgvs_Protein; "pyhgvs Protein"
+    lstring change_type; "change type"
+    string _mouseOver; "mouse over field hidden"
+    )
```

Comment:
![screen shot 2017-07-27 at 10 44 38 am](https://user-images.githubusercontent.com/4503616/28684335-d849171c-72b8-11e7-8ecb-8b69412f90f5.png)
The red circle is Github complaining that there's no newline at the end of the file. please add one.